client/my-sites/site-settings/site-tools/index.jsx

```diff
@@ -0,0 +1,148 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { some } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import CompactCard from 'components/card/compact';
+import DeleteSiteWarningDialog from 'my-sites/site-settings/delete-site-warning-dialog';
+import config from 'config';
+import { tracks } from 'lib/analytics';
+import { localize } from 'i18n-calypso';
+import SectionHeader from 'components/section-header';
+
+const trackDeleteSiteOption = ( option ) => {
+	tracks.recordEvent( 'calypso_settings_delete_site_options', {
+		option: option
+	} );
+};
+
+class SiteTools extends Component {
+	static propTypes = {
+		sitePurchases: PropTypes.array.isRequired,
+		hasLoadedSitePurchasesFromServer: PropTypes.bool.isRequired,
+		site: PropTypes.object.isRequired
+	}
+
+	state = {
+		showDialog: false,
+		showStartOverDialog: false,
+	}
+
+	render() {
+		const { translate } = this.props;
+
+		const selectedSite = this.props.site;
+		const changeAddressLink = `/domains/manage/${ selectedSite.slug }`;
+		const themeSetupLink = `/settings/theme-setup/${ selectedSite.slug }`;
+		const startOverLink = `/settings/start-over/${ selectedSite.slug }`;
+		const deleteSiteLink = `/settings/delete-site/${ selectedSite.slug }`;
+		let changeAddressLinkText = translate( 'Register a new domain or change your site\'s address.' );
+		const themeSetupLinkText = translate( 'Automatically make your site look like your theme\'s demo.' );
+		const strings = {
+			changeSiteAddress: translate( 'Change your site address' ),
+			themeSetup: translate( 'Theme setup' ),
+			startOver: translate( 'Delete your content' ),
+			deleteSite: translate( 'Delete your site' )
+		};
+
+		if ( ! this.props.hasLoadedSitePurchasesFromServer ) {
+			return null;
+		}
+
+		if ( ! config.isEnabled( 'upgrades/domain-search' ) ) {
+			changeAddressLinkText = translate( 'Change your site address.' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **39** times:
`translate( 'Change Site Address' )` *ES Score: 9*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/modify/settings-delete-options)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).